### ISSUE TITLE
SSF for Android is very stable Version - Don't give up on that!

### ISSUE THREAD

**Issue by EX-Soldat** [shimazzz/SEGASaturnEmulator-SSF#24]
You are close to have the best Saturn Emulator on Android out. Already love this app and use it recently often. For me the SSF was the only Emulator what ever could come close to a Saturn-Experience on Windows-PC's and I didn't hesitate to download and try around with the Android-app when you made it available. Please don't stop tuning it. The code-informations on the left screen should disappear while playing or should have some Options at least to disable them. I will report more soon and gonna test it more with many other Games- Thanks for your hard work Shima-san 🙏

**Comment by EX-Soldat** [comment]
> You are close to have the best Saturn Emulator on Android out. Already love this app and use it recently often. For me the SSF was the only Emulator what ever could come close to a Saturn-Experience on Windows-PC's and I didn't hesitate to download and try around with the Android-app when you made it available. Please don't stop tuning it. The code-informations on the left screen should disappear while playing or should have some Options at least to disable them. I will report more soon and gonna test it more with many other Games- Thanks for your hard work Shima-san 🙏



My Report so far after some testing.

- there is no option to change the disk. Some multiple disk Games require it to open the tray and close it after changing the disk.

- an Option to disable the running code-informations on the left as well the Versionnumber on the down left.

Games not running with issues report:

- Last Bronx NTSC us Version - hanging while loading. No Screen.

- Three dirty Dwarfes - Menu issues and Button lags. Crashing while loading into the 1st Stage.

Games got to work after tweaking some Options:

- Toshinden URA - set idle loop to off.

**Comment by vjunior613** [comment]
I Agree with all that you said. The SSF is by far the best emulator to play Saturn on Windows, and the SSF Android Version, besides still very early in development, was already playing games better than the YabaSanshiro2 or any of those ports of Yabause code. Continue with the project, please, don't give it up!!

**Comment by SRC267** [comment]
> I Agree with all that you said. The SSF is by far the best emulator to play Saturn on Windows, and the SSF Android Version, besides still very early in development, was already playing games better than the YabaSanshiro2 or any of those ports of Yabause code. Continue with the project, please, don't give it up!!

Right now the new Saturn emu by Robert Broglia is the best. It's based on Mednafen, and was re-released a few weeks ago, on GitHub or Playstore (paid)

**Comment by vjunior613** [comment]
> > I Agree with all that you said. The SSF is by far the best emulator to play Saturn on Windows, and the SSF Android Version, besides still very early in development, was already playing games better than the YabaSanshiro2 or any of those ports of Yabause code. Continue with the project, please, don't give it up!!
> 
> Right now the new Saturn emu by Robert Broglia is the best. It's based on Mednafen, and was re-released a few weeks ago, on GitHub or Playstore (paid)

I disagree, although the android SSF is still in BETA, it already runs much better than any other Sega Saturn emulator, at least on my Moto G52, 99% of the games run at 100%, while Saturn.emu is quite slow. Really, the only thing that is bothering me about SSF are these red bars and numbers on the sides of the screen, which I would really like to remove, and for the games folder to be removed from the android folder, but otherwise, the performance the emulator is very good!!!

**Comment by SRC267** [comment]
> I disagree, although the android SSF is still in BETA, it already runs much better than any other Sega Saturn emulator, at least on my Moto G52, 99% of the games run at 100%, while Saturn.emu is quite slow. Really, the only thing that is bothering me about SSF are these red bars and numbers on the sides of the screen, which I would really like to remove, and for the games folder to be removed from the android folder, but otherwise, the performance the emulator is very good!!!

I'll have to try SSF again, I opened it on my android and it was just a black screen with small text in the left corner. I didn't check how to set it up, didn't see a menu button.
